test/cypress/cypress.config.js

```diff
@@ -1,9 +1,57 @@
+const path = require('path');
+require('dotenv').config({ path: path.resolve(__dirname, '../../server/.env') });
+
 const { defineConfig } = require("cypress");
+const { Client } = require('pg');
 
 module.exports = defineConfig({
+  component: {
+    devServer: {
+      framework: "vue-cli",
+      bundler: "webpack",
+    },
+  },
+
   e2e: {
     setupNodeEvents(on, config) {
-      // implement node event listeners here
+      on("task", {
```

Comment:
Love this. These are the tricks that can take hours to learn and there is no guarantee you'll find "this" solution when hustling to make something work.